cmd/juju/commands/common.go

```diff
@@ -226,12 +226,10 @@ var newCharmStoreClient = func() (*csClient, error) {
 func newHTTPClient() (*cookiejar.Jar, *http.Client, error) {
 	cookieFile := path.Join(utils.Home(), ".go-cookies")
```

Comment:
I'd probably do more of a direct backport and use cookieFile here
(as defined in master), and Trace any returned error.